src/gpu/d3d12/SDL_gpu_d3d12.c

```diff
@@ -3364,12 +3575,17 @@ static SDL_GPUTexture *D3D12_CreateTexture(
     }
 
     container->debugName = NULL;
+    if (SDL_HasProperty(createinfo->props, SDL_PROP_GPU_CREATETEXTURE_NAME_STRING)) {
+        container->debugName = SDL_strdup(SDL_GetStringProperty(createinfo->props, SDL_PROP_GPU_CREATETEXTURE_NAME_STRING, NULL));
```

Comment:
Does SDL_strdup() properly handle NULL? I'm pretty sure I've seen this crash in some C runtimes

Reply:
We're already checking HasProperty above, so it should never return NULL right?

Reply:
Right, multi-threading aside.